lib/git_reflow/git_server/pull_request.rb

```diff
@@ -3,6 +3,16 @@ module GitServer
     class PullRequest
       attr_accessor :description, :html_url, :feature_branch_name, :base_branch_name, :build_status, :source_object, :number
 
+      LGTM_REGEX = /(?i-mx:lgtm|looks good to me|[:\\+1:]|:thumbsup:|:shipit:)/
```

Comment:
We wanted to keep the default as the current behavior, of all approvers must approve, which we defined as `""`.  We can make it `"-1"` if that seems better to you.  However we can't prevent people from setting the config value to `""`, so we need to handle that case regardless.